### PR TITLE
Translation into Brazilian Portuguese

### PR DESCRIPTION
**Pull request by mykael90**
Firstly, I would like to thank you for the excellent work done. Here is a small contribution: Translation into Brazilian Portuguese to solve issue #18 
Sorry if some project guidelines were not followed.

### CODE REVIEW

**Comment by camelaissani** [comment]
@mykael90 thank you for your kind message and your PR!

File: services/emailer/src/emailparts/contents/invoice/pt-BR/subject.ejs [hunk 1]
```diff
@@ -1 +1,2 @@
-Rent invoice of <%- _.formatTerm(period, tenant.contract.lease.timeRange) %>
\ No newline at end of file
+Cobrança de aluguel de <%- _.formatTerm(period, tenant.contract.lease.timeRange)
+%>
```
Comment (camelaissani) [inline services/emailer/src/emailparts/contents/invoice/pt-BR/subject.ejs:2]: Could you remove the line break please?

File: services/emailer/src/emailparts/contents/rentcall/pt-BR/subject.ejs [hunk 2]
```diff
@@ -1 +1,2 @@
-Payment notice of <%- _.formatTerm(period, tenant.contract.lease.timeRange) %>
\ No newline at end of file
+Aviso para pagamento de aluguel <%- _.formatTerm(period,
+tenant.contract.lease.timeRange) %>
```
Comment (camelaissani) [inline services/emailer/src/emailparts/contents/rentcall/pt-BR/subject.ejs:2]: Same here

File: services/emailer/src/emailparts/contents/rentcall_last_reminder/pt-BR/subject.ejs [hunk 3]
```diff
@@ -1 +1,2 @@
-Last reminder before EVICTION request: unpaid rent of <%- _.formatTerm(period, tenant.contract.lease.timeRange) %>
\ No newline at end of file
+ÚLTIMO AVISO: Pendência de pagamento referente a aluguéis de <%-
+_.formatTerm(period, tenant.contract.lease.timeRange) %>
```
Comment (camelaissani) [inline services/emailer/src/emailparts/contents/rentcall_last_reminder/pt-BR/subject.ejs:2]: Same here

**Comment by camelaissani** [comment]
I updated the issue #18 with 2 other files to translate in case you have time

**Comment by mykael90** [comment]
With sure! I did the updates requested in the last commit

File: services/emailer/src/emailparts/contents/rentcall/pt-BR/subject.ejs (hunk 2 above)
Comment (camelaissani) [reply services/emailer/src/emailparts/contents/rentcall/pt-BR/subject.ejs:2]: On this one the line break is still here please.

**Comment by mykael90** [comment]
Now I thi it's correct. Lint was being applied on save.

**Review by camelaissani — APPROVED** [review]

**Pull request merged**